### PR TITLE
Updated ReadMe, Corrected some Grammatical Mistakes

### PR DESCRIPTION
**Pull request by gargarchit**
Hi
I have corrected some Grammatical Mistakes in ReadMe, Hence it now looking good and easier to read.
Please Merge this

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by gargarchit** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @gargarchit. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @gargarchit, here is your [new Magento instance](https://pr-24864.instances.magento-community.engineering).
Admin access: [https://pr-24864.instances.magento-community.engineering/admin](https://pr-24864.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: README.md [hunk 1]
```diff
@@ -30,14 +30,14 @@ To suggest documentation improvements, click [here][4].
 [4]: https://devdocs.magento.com
 
 <h3>Community Maintainers</h3>
-The members of this team have been recognized for their outstanding commitment to maintaining and improving Magento. Magento has granted them permission to accept, merge, and reject pull requests, as well as review issues, and thanks these Community Maintainers for their valuable contributions.
+The members of this team have been recognized for their outstanding commitment to maintaining and improving Magento. Magento has granted them permission to accept, merge, and reject pull requests, as well as review issues, and thanks to these Community Maintainers for their valuable contributions.
```
Comment (jeff-matthews) [inline README.md]: The original copy is correct. Please remove "to".

Magento has granted and [Magento] thanks...

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gargarchit) [reply README.md]: Thank you @jeff-matthews for Reviewing this 
I have updated changes please have a look

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-6006](https://jira.corp.magento.com/browse/ENGCOM-6006) has been created to process this Pull Request
:eight_spoked_asterisk: @jeff-matthews, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@gargarchit thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by gargarchit** [comment]
Thank You @jeff-matthews  @magento-engcom-team 
Thank You for Inviting me as Member of Community

**Comment by gargarchit** [comment]
@jeff-matthews, could you please add one of the labels to the Pull Request as mentioned above by @magento-engcom-team

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**